examples/minimal.rs

```diff
@@ -170,7 +166,11 @@ async fn start_node(
     contacts: Vec<SocketAddr>,
     ip: Option<IpAddr>,
     base_port: Option<u16>,
-) -> SocketAddr {
+) -> (SocketAddr, JoinHandle<()>) {
+    if !first && contacts.is_empty() {
+        panic!("non-first node shall have non-empty contacts to bootstrap with!");
```

Comment:
Maybe add this also to the structopt thing, so we get nicer output? Something like this:
```rust
#[structopt(..., required_unless = "first")]
bootstrap_contacts: Vec<SocketAddr>,
```
?

Reply:
required_unless: https://docs.rs/clap/2.33.3/clap/struct.Arg.html#method.required_unless